src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import tensorflow as tf
+from .compressor import Pruner
+
+__all__ = [ 'LevelPruner', 'AGP_Pruner', 'SensitivityPruner' ]
+
+_logger = logging.getLogger(__name__)
+
+
+class LevelPruner(Pruner):
+    def __init__(self, config_list):
+        """
+            Configure Args:
+                sparsity
+        """
+        super().__init__(config_list)
+
+    def calc_mask(self, weight, config, **kwargs):
+        threshold = tf.contrib.distributions.percentile(tf.abs(weight), config['sparsity'] * 100)
+        return tf.cast(tf.math.greater(tf.abs(weight), threshold), weight.dtype)
+
+
+class AGP_Pruner(Pruner):
+    """
+    An automated gradual pruning algorithm that prunes the smallest magnitude 
+    weights to achieve a preset level of network sparsity.
+
+    Michael Zhu and Suyog Gupta, "To prune, or not to prune: exploring the
+    efficacy of pruning for model compression", 2017 NIPS Workshop on Machine
+    Learning of Phones and other Consumer Devices,
+    https://arxiv.org/pdf/1710.01878.pdf
+    """
+    def __init__(self, config_list):
+        """
+            Configure Args
```

Comment:
could you follow the docstring format in other tuner...

Reply:
I found compressors' docstring format is different from other modules (like tuner).
But I don't know what format we are using exactly.
@SparkSnail Any idea?